test/moment-range.js

```diff
@@ -113,11 +116,29 @@ describe('DateRange', function() {
     });
 
     it('should allow initialization with undefined arguments', function() {
-      var dr = moment.range(undefined, undefined);
+      var dr = moment.range(undefined, undefined, undefined);
 
       moment.isMoment(dr.start).should.be.true;
       moment.isMoment(dr.end).should.be.true;
     });
+    
+    it('should allow initialization with options', function() {
+      var dr = moment.range(sStart, sEnd, options);
+
+      moment.isMoment(dr.start).should.be.true;
+      moment.isMoment(dr.end).should.be.true;
+      dr.options.should.be.type('object');
+      
+      dr.options.should.have.property('exists');
+    });
+
+    it('should allow initialization with options as second option', function() {
+      dr = moment.range(m1, options);
+      
+      moment.isMoment(dr.end).should.be.true;
+      dr.options.exists.should.be.true;
+    });
+
   });
```

Comment:
add a test that subtraction now returns a new object instead of modifying the original